libs/application/template-api-modules/src/lib/modules/templates/institution-application/emailGenerators/applicationOverviewTemplate.ts

```diff
@@ -0,0 +1,222 @@
+import { Application } from '@island.is/application/core'
+import { messages } from '@island.is/application/templates/institution-application'
+import get from 'lodash/get'
+import { dedent } from 'ts-dedent'
+
+export const applicationOverviewTemplate = (
+  application: Application,
+): string => {
+  const institutionName = get(application.answers, 'applicant.institution')
+
+  const contactName = get(application.answers, 'contact.name')
```

Comment:
We have an util for this in the application core that you can use https://github.com/island-is/island.is/blob/main/libs/application/core/src/lib/formUtils.ts#L36